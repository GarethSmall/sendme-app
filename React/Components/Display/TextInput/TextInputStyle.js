// @flow
import { StyleSheet } from 'react-native';
import AppTheme from '../../../AppTheme/AppTheme';

const TextInputStyle = StyleSheet.create({
  input: {
    height: 45,
    alignSelf: 'stretch',
    marginBottom: AppTheme.largeSize,
    padding: AppTheme.mediumSize,
    backgroundColor: AppTheme.whiteColor,
    borderRadius: AppTheme.smallSize,
  },
});

export default TextInputStyle;