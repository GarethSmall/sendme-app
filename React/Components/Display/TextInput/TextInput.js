// @flow
import React, { PureComponent } from 'react';
import TextInputStyle from './TextInputStyle';
import ReactNative from 'react-native';

export default class TextInput extends PureComponent {
  render() {
    const { style, ...other } = this.props;
    return <ReactNative.TextInput {...other} style={[TextInputStyle.input, style]} />;
  }
}
