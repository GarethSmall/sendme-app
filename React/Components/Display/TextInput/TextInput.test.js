// @flow
import React from 'react';
import TextInput from './';
const { describe, test, expect } = global;
import renderer from 'react-test-renderer';

describe('TextInput ', () => {
  test('should render', () => {
    const tree = renderer.create(
      <TextInput  />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
