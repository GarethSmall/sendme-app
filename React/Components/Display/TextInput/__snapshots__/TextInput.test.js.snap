// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TextInput  should render 1`] = `
<TextInput
  allowFontScaling={true}
  style={
    Array [
      Object {
        "alignSelf": "stretch",
        "backgroundColor": "#fff",
        "borderRadius": 4,
        "height": 45,
        "marginBottom": 20,
        "padding": 9,
      },
      undefined,
    ]
  }
/>
`;
